projects/fusion-e2e/tests/components/alert/alert-component.ts

```diff
@@ -1,45 +1,59 @@
-import {Page} from '@playwright/test';
 import {AlertTestIdModifiers} from '@ironsource/fusion-ui/entities';
 import {getTestId, getTestIdSelector} from '../../global/utils';
-
-export class AlertComponent {
-    readonly page: Page;
-
-    constructor(page: Page) {
-        this.page = page;
+import {BaseComponent} from '../base-component';
+import {BaseElement} from '../../behavior';
+import {ButtonComponent} from '../button/button-component';
+
+export class AlertComponent extends BaseComponent {
+    private wrapperElement: BaseElement;
+    private messageElement: BaseElement;
+    private titleElement: BaseElement;
```

Comment:
All of the component class properties should inherit from a designated class that is not base element. For example title element should implement from StaticText class. Same note applies to wrapperElement and messageElement

Reply:
Done